CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
 
+## [2.9.0] - 2018-06-28
```

Comment:
It is best to leave the versioning and dates to the maintainers, leaving only your summary of the change and the header for the impact under `## [Unreleased]`. There are several reasons for this:
- we date the release not the contribution
- maintainers may disagree on what the "next" version is based on the impact of the change (in this case its fine)
- PRs are not a FIFO queue and as such merge order is not guaranteed, if another PR merges you will need to rebase and fixup.
- there is no SLA on PR reviews